### PR TITLE
Uses "read-from-string" to parse the input as a lisp list

### PR DESCRIPTION
**Pull request by weslleymberg**
An step foward on issue #18

### CODE REVIEW

File: web/webapp.lisp [hunk 1]
```diff
@@ -42,9 +42,10 @@
...
   "Translate string to a list expression"
-  (mapcar (lambda (x)
-            (intern (string-upcase x) :lisp-inference))
-          (str:words string)))
+  (if (and (str:starts-with-p "(" string)
+           (str:ends-with-p ")" string))
+      (read-from-string string)
+      (read-from-string (str:concat "(" string ")"))))
 
 (defun trim-paren (string)
```
Comment (ryukinix) [inline web/webapp.lisp]: poderia remover essa função?

Comment (weslleymberg) [reply web/webapp.lisp]: Adicionei o commit pra remover.

Comment (ryukinix) [reply web/webapp.lisp]: Ótimo! Agora precisamos remover a funcall dela no método render para o tipo `truth-table`

Comment (ryukinix) [reply web/webapp.lisp]: Aqui https://github.com/ryukinix/lisp-inference/blob/72cea2b76aa42628e6dcddda85582244e0805bdf/web/webapp.lisp#L82

Comment (weslleymberg) [reply web/webapp.lisp]: Done!

File: web/webapp.lisp [hunk 2]
```diff
@@ -79,7 +79,7 @@
                              (update-proposition table prop)))
       (:input :type "text"
               :name "prop"
-              :placeholder (trim-paren (prop table)))
+              :placeholder (prop table))
```
Comment (ryukinix) [inline web/webapp.lisp]: Excelente!

**Pull request merged**